html-boilerplate/css/index.css

```diff
@@ -0,0 +1,26 @@
+
+body {background-color: powderblue;
+      font-family: verdana;
+}	
+p {font-family: "Sofia", sans-serif; font-size: 20px; margin: 5px; padding: 5px;}
```

Comment:
Generally when writing your css rules, you want to put each one on a new line so its easier to read and organize, so like: 
```css
p {
  font-family: "Sofia", sans-serif;
  font-size: 20px;
  margin: 5px;
  padding: 5px;
}
```